examples/custom_distribution.ipynb

```diff
@@ -0,0 +1,425 @@
+{
```

Comment:
<p>is the data public? Can it be added to the <code>data</code>  folder and read with the try except described at https://github.com/pymc-devs/pymc3/wiki/PyMC3-Jupyter-Notebook-Style-Guide. I also noticed the notebook is on <code>examples/</code>  folder, I think it should go on <code>examples/pymc3_howtos</code>  one (as well as eventually being added to the table of contents so it appears on the web)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc-examples/pull/185/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Yes this data is sourced from https://www.mass.gov/info-details/archive-of-covid-19-cases-in-massachusetts. I'll add it the csv I'm using to the `data` folder and move this file to `examples/pymc3_howtos`

Reply:
I'll also follow the `try...except` clause recommended in the style guide